app/src/main/java/fr/free/nrw/commons/notification/NotificationClient.kt

```diff
@@ -21,43 +22,55 @@ class NotificationClient
         @param:Named(NetworkingModule.NAMED_COMMONS_CSRF) private val csrfTokenClient: CsrfTokenClient,
         private val service: NotificationInterface,
     ) {
-        fun getNotifications(archived: Boolean): Single<List<Notification>> =
+    fun getNotifications(archived: Boolean): Single<List<Notification>> =
```

Comment:
What is the purpose of this change of a large number of lines? If it is indentation, better do it in a separate pull request, to make the changes in this one easier to spot. Thanks!

Reply:
I think there might be some Android studio version format issues for the indentation, there was why it changed. There was weird, I did not notice that when I pushing.

Reply:
Do I still need to change it back to the old one? I afraid of making more mess on it.

Reply:
If you are afraid of messing things up, the safest might be to create a new branch from main, and apply your changes in that new branch, then send a new pull request based on it.

Thanks a lot! 🙂

Reply:
Got that, I gonna create a new pull request for that.
Cheers